_includes/configuration/toolbar.md

```diff
@@ -48,6 +48,25 @@ tinymce.init({
 });
 ```
 
+### Enabling Sticky Toolbars
+
+A Sticky Toolbar (or Docking Toolbar), docks the toolbar to the top of the screen when scrolling down a web page until the editor is no longer visible. To enable Stick toolbars declare `toolbar_sticky` as `true` in the `tinymce.init` script. To see a demo of sticky toolbar, see the [Full featured demo]({{ site.baseurl }}/demo/full-featured/).
```

Comment:
I think we need to probably a small note or something to clarify it's the `toolbar` and `menubar` that gets docked, not just the toolbar.

Reply:
Updated. 'A Sticky Toolbar (or Docking Toolbar), docks the toolbar and the menu ...'